getting-started/CONCEPTS/AI-SYSTEM.md

```diff
@@ -1,7 +1,8 @@
 # The AI System
 
-The AI system is responsible for moderating input and output, generating plans, and executing them. It can be used free standing or routed to by the Application object. This system is encapsulated in the `AI` class. It is made of three components, the moderator, the planner, and actions.
+The AI system is responsible for moderating input and output, generating plans, and executing them. It can be used free standing or routed to by the Application object. This system is encapsulated in the `AI` class. It is made of four components, the moderator, the planner, actions, and data sources.
 
 1. [Moderator](./MODERATOR.md)
 2. [Planner](./PLANNER.md)
 3. [Actions](./ACTIONS.md)
+4. [Data Sources](./DATA-SOURCES.md)
```

Comment:
I don't think data sources belong in this section.

Technically, data sources is part of the prompt management system. The `AI system` here refers to the AI class which virtually only knows about actions, planners and actions. And the data sources component is part of the action planner (maybe assisstants planner too?) through the prompt manager. 

I think it should be moved to either the `ActionPlanner` or the `Prompt Manager` concept.

Reply:
Ah I assumed you wanted it in this section based on the issue breakdown, but I think it makes more sense in the Prompts section instead as well